src/main/resources/internal-admin-api.yaml

```diff
@@ -1102,7 +1102,23 @@ paths:
           schema:
             $ref: '#/definitions/JsonWebKeySet'
 
-  /testing-support/accounts:
+  /testing-support/accounts/{accountId:.+}:
+    get:
+      operationId: "getAccount"
+      description: "Fetch a test account via ID"
+      parameters:
+        - in: path
+          name: accountId
+          type: string
+          required: true
+          description: "The id of the user to invite"
```

Comment:
it's the uid right?

Reply:
Yeah, I don't understand why its called "account" here but just conforming to the rest of it